ddtrace/tracer/option.go

```diff
@@ -562,6 +588,21 @@ func newConfig(opts ...StartOption) *config {
 	// This allows persisting the initial value of globalTags for future resets and updates.
 	globalTagsOrigin := c.globalTags.cfgOrigin
 	c.initGlobalTags(c.globalTags.get(), globalTagsOrigin)
+
+	// TODO: change the name once APM Platform RFC is approved
+	if internal.BoolEnv("DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED", false) {
```

Comment:
Optional -- but I think it'd be best if all those environment variable names were exposed from constants somewhere...

Reply:
From what I see in the tracer the opposite is very true so I will prefer normalization here